Add list operation example

koertho opened the pull request:
This PR adds an example for list operations to the dca docs.

koertho committed:
add list operation example

fritzmg reviewed (changes requested):
Unfortunately I don't think this is a good example. This might give the false impression that this prevents non-admins from deleting records. However it only hides the button - and does not prevent the actual action.

koertho committed:
updated example

koertho commented:
Ok, I changed the example to a custom button and added additional explanation.
I also enhanced the code with Backend class instead of Controller class and the usage of sprintf.

fritzmg commented on docs/dev/reference/dca/callbacks.md:
```diff
@@ -599,6 +599,64 @@ an additional command check via load_callback).
+This example hide a custom operation button if the user is not allowed to use it.    
+
+Attention: this won't disable the operation itself, it only hides the button!
+To disable the operation, you need to check for the permission additional before it is execution, for example in the operation backend module code.
```
```suggestion
Attention: this won't disable the operation itself, it only hides the button!
To disable the operation, you need to check for the permission additionally 
before its execution, for example in the operation code or a `config.onload` callback.
```

koertho committed:
use suggestion

Co-authored-by: Fritz Michael Gschwantner <fmg@inspiredminds.at>

fritzmg commented:
Thank you @koertho :)
